extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -149,12 +158,26 @@
   position: relative !important;
 }
 
+.i-amphtml-story-interactive-slider-input::-moz-range-thumb {
+  width: var(--thumb-size) !important;
+  height: var(--thumb-size) !important;
+  background: var(--interactive-accent-color) !important;
+  border-radius: 50% !important;
+  border: none !important;
+}
+
 .i-amphtml-story-interactive-mid-selection .i-amphtml-story-interactive-slider-input::-webkit-slider-thumb {
   transform: rotate(45deg) !important;
   box-shadow: 0em 0em 0.5em rgba(0, 0, 0, 0.25),
     0em 0em 0em 0.07em var(--i-amphtml-interactive-option-background-color) !important;
   border-top-left-radius: 0% !important;
-  z-index: 2 !important;
+}
+
+.i-amphtml-story-interactive-mid-selection .i-amphtml-story-interactive-slider-input::-moz-range-thumb {
```

Comment:
Same comment as above, maybe this selector can be added to the `.i-amphtml-story-interactive-mid-selection .i-amphtml-story-interactive-slider-input::-webkit-slider-thumb` ruleset.